### DISCUSSION TITLE
Relations to models with only translations are hard to maintain

### DISCUSSION THREAD

**Discussion by viters** [directus/directus#12035] [Feature Requests]
If I have a collection (model), which only has `translations`, such collection is hard to use in relation, because I must maintain the relation basing on IDs. It would be useful to select a default language and use translations (or part of it) in display template. Other solution would be clickable translation's details in display template like relations are right now.

![image](https://user-images.githubusercontent.com/13100466/157271001-d972211c-8321-4327-8292-e2071baae79a.png)

**Comment by rijkvanzanten** [comment]
What are we looking at in this screenshot? Is that a drawer that shows the current items in `<collection>_translations`?

**Reply by viters** [reply]
It's a drawer for M2M / O2M / M2O relations for collections which have only translations.

**Reply by rijkvanzanten** [reply]
You can setup the displays for that related collection, and they'll show up in the drawer as well 🙂 

Those drawers use your current configuration for the related collection, so you could also open the collection directly, setup the layout to your ideal state, and it'll look like that in drawers as well 👍🏻

**Reply by viters** [reply]
Yeah, I know. You can modify drawer, but you cannot use `translations` in displays in effective way.

**Comment by viters** [comment]
Can be closed after recent changes to translations display view. You can set default language and show deep translation field value.

**Comment by viters** [comment]
@rijkvanzanten 

I see one more area with translations that I think were left after changes with display and default language.

When I have M2M relationship, I cannot use `translation[field]` as label.

I have `navigation` collection that holds `navigationItems`. Navigation items are translatable and have `translations.label`:

![image](https://user-images.githubusercontent.com/13100466/199592879-15a41e57-a6e0-45af-92db-e96ba9a0d762.png)

To properly show meaningful stuff I would need to use:

![image](https://user-images.githubusercontent.com/13100466/199593105-f10a0118-253c-42e3-baaf-87964b52c9cc.png)

Although that results in:

![image](https://user-images.githubusercontent.com/13100466/199593162-c16f390e-81c2-4980-8730-ce36c3a4f4e5.png)

**Reply by rijkvanzanten** [reply]
> To properly show meaningful stuff I would need to use:

Try using the parent `Translations` field there with the translations display configured on that `Translations` field. What you've selected now would result in an array of strings coming from the API, which isn't gracefully rendered in app at the moment (https://github.com/directus/directus/issues/12271)

**Reply by viters** [reply]
Amazing, it works!
